sources/PTYTextView.h

```diff
@@ -589,6 +593,9 @@ typedef void (^PTYTextViewDrawingHookBlock)(iTermTextDrawingHelper *);
 
 - (void)setTransparencyAffectsOnlyDefaultBackgroundColor:(BOOL)value;
 
+- (IBAction)selectCurrentCommand:(id)sender;
```

Comment:
Most IBActions aren't exposed in the header file. IB is able to find them in .m files. Are these needed for a particular purpose?

Reply:
Of course, if you wire up those actions using IB, they can stay in the implementation file. If however you need to access those methods from another class, they need to be visible in the header. That's why I've exposed them.